modules/gdscript/gdscript_editor.cpp

```diff
@@ -1067,6 +1065,33 @@ static void _find_identifiers(GDScriptParser::CompletionContext &p_context, bool
 		kw++;
 	}
 
+	static const char *_keywords_with_space[] = {
+		"and", "in", "not", "or", "as", "class", "extends", "is", "func", "signal", "await",
+		"const", "enum", "static", "var", "if", "elif", "else", "for", "match", "while",
+		0
+	};
+
+	const char **kws = _keywords_with_space;
+	while (*kws) {
+		ScriptCodeCompletionOption option(*kws, ScriptCodeCompletionOption::KIND_PLAIN_TEXT);
+		option.insert_text += " ";
+		r_result.insert(option.display, option);
+		kws++;
+	}
+
+	static const char *_keywords_with_args[] = {
+		"assert", "preload",
+		0
+	};
+
+	const char **kwa = _keywords_with_args;
+	while (*kwa) {
+		ScriptCodeCompletionOption option(*kwa, ScriptCodeCompletionOption::KIND_PLAIN_TEXT);
```

Comment:
This can be `KIND_FUNCTION` since they act like functions.

Reply:
The only thing that would change is the icon. Just confirm that's what you want. 🙂 
https://github.com/godotengine/godot/blob/68f24175286ec26a4a9a1566d8e25a1fbe6f4caa/editor/code_editor.cpp#L883-L888

Reply:
I missed this, but yes, I prefer they have the function icon.

Reply:
Fixed.